lib/actions/addLinkToProjectAction.js

```diff
@@ -2,12 +2,24 @@ const AddLinkToProjectResponse = require('./AddLinkToProjectResponse')
 
 class AddLinkToProjectAction {
 
-  async execute([label, link]){ 
+  parseParams(string) {
+    let stringToArray = string.split(' ')
```

Comment:
I’m struggling with the naming of these variables.  I love that you return an object 💪